### DISCUSSION TITLE
Per user editor selector missing

### DISCUSSION THREAD

**Discussion by N6REJ** [joomla/joomla-cms#37028] [Q&A] [answered]
Was the ability to select which editor to use on a per user basis intentionally removed?
It's extremely useful for sites that have multiple levels of admins.
For example we have several clients that aren't competent enough to use JCE so we provide a stripped down editor for them in tinymce
![image](https://user-images.githubusercontent.com/1850089/153781547-bb973969-cc25-4ad8-856b-bafdd833937b.png)

**Comment by brianteeman** [comment]
the problem is either in your language overrides or in the language (en-US?) file as the string is not correct but it is correct in core.

![image](https://user-images.githubusercontent.com/1296369/153822548-d4ee3f7b-073b-43e3-8cf6-077ba4ff54d1.png)

**Comment by brianteeman** [comment]
Confirmed it is a bug in the en-US language pack

![image](https://user-images.githubusercontent.com/1296369/153919424-b930e5f4-5797-445b-8559-3181985902fa.png)

Note that in just that one file there are strings in multiple languages!!

**Comment by particthistle** [comment, marked as answer]
Resolving now in Crowdin. Next en-US language pack (and some other packs also affected by what's been identified) will have this corrected.

Details on other items affected: https://github.com/joomla/core-translations/issues/185
